MarketExecute.php

```diff
@@ -153,7 +153,7 @@ function DefaultErrorHandler($errno, $errstr) {}
 require("./Database.php");
 require("./Product.php");
 require("./ProductsExplorer.php");
-require('./vendor/autoload.php');
+#require('./vendor/autoload.php');
```

Comment:
This is importing AWS S3 SDK set up in the server environment. It's really important.

Maybe add a comment about this so it's less confusing.